src/app/utils/ServiceUtils.ts

```diff
@@ -295,6 +295,23 @@ const ServiceUtils = {
 		};
 		return (await dmChannel.awaitMessages(replyOptions)).first().attachments.first();
 	},
+	
+	sendOutErrorMessage: async (ctx: CommandContext): Promise<any> => {
```

Comment:
In a future update, can we include the support server link here?

Reply:
already there ser on line 306 :)